### ISSUE TITLE
Add source map support

### ISSUE THREAD

**Issue by zone117x** [blockstack/blockstack-app#2084]
Errors are difficult to debug when given a stack trace because there's no source map support. 

Here's an example stack trace where source map support would have made it much easier to debug an issue:

![image](https://user-images.githubusercontent.com/1447546/146239239-5e96839d-1057-4ca3-92fb-647dfeab40b0.png)

**Comment by andresgalante** [comment]
+1, this will also help for Sentry errors. I am hoping to add this to our cue on Q1

**Comment by kyranjamie** [comment]
Yep, indeed. I have a PR that never got merged https://github.com/hirosystems/stacks-wallet-web/pull/1876

Sourcemaps are a little trickier for extensions, as Sentry can't just go and fetch them. We need to upload them on release.

**Comment by zone117x** [comment]
This issue has come up again. QA is running into errors that are potentially blocking/delaying another API release because the there's no (useful) error logging or source maps:
![image](https://user-images.githubusercontent.com/1447546/151838431-c0d2c288-6cb5-4688-9103-9b4f56598094.png)

```
index.js:150788 TypeError: Network request failed
    at XMLHttpRequest.xhr.onerror (index.js:83310:16)
onError @ index.js:150788
```

**Comment by andresgalante** [comment]
Thanks Matt for bringing it up. This one is in the radar for @He1DAr as he improves our Sentry implementation
